libs/boringssl/Makefile

```diff
@@ -0,0 +1,52 @@
+# SPDX-License-Identifier: GPL-2.0-only
```

Comment:
@jefferyto Could I clarify what the guidelines for the License/Copyright are here? Is this specific to the package or the stuff that gets packaged? I saw different formats being followed across packages, is there are preferred way?

Reply:
I'm not really sure if there are guidelines on specifying the license of the makefile/package itself (or what those guidelines may be). I had assumed all contributions followed the same license as the repo.

Maybe @neheb or @BKPepe would know more about this?